src/Relation/Binary/Construct/Interior/Symmetric.agda

```diff
@@ -0,0 +1,91 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Symmetric interior of a binary relation
```

Comment:
Could we have a little bit more description about what an `interior` actually is?

Reply:
Try (and edit as you see fit, @laMudri ): 

"Just as the `P`-closure of a relation `R` for some property `P` is the *least* relation `S` containing `R` such that `P S`, the `P`-*interior* is the *dual* notion, vs. the *greatest* relation `S` *contained in* `R` such that `P S`, for any suitable `P`. Here, we take `P` to be `Symmetric`."

Or some such...

For a citation, you could look at the treatment of 'intersection-closed' and 'union-closed' families of predicates/relations defined in [Paul M. Cohn's "Universal Algebra" (1965)](https://link.springer.com/book/10.1007/978-94-009-8399-1) and their application to defining closures and interiors, and inductive/coinductive definitions generally (see also Peter Aczel's article on inductive definitions in the Handbook of Mathematical Logic (1977) (ISBN: 9780720422856)). NB impredicative definitions! 

Being `Symmetric` is, IIRC, an example of a property which is both intersection- and union-closed, and hence has a corresponding closure and interior, as here.

Reply:
On second thoughts, maybe I should (have) move(d) this comment to the issue, rather than pollute the specific PR addressing it? Will do that, and then you can cherry-pick from there...